nss-day-cohort-48/Rare-API-Django-Wandering-Wizards
# added image to post and author

#### Changes Made

1. Added file `filename` to `directoryname` directory.
2. Modified file `filename` to include `functionalityname` and functionality.

#### Steps to Review
1. Checkout to this branch locally
   ```
   git fetch --all
   git checkout ajw-adding-image-upload
   ```

2. Ensure the backend and client are both running.

4. Test app functionality.
   
   > Example: When user does BLANK, then BLANK should happen.
   > 

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
